src/constants/item/colors.js

```diff
@@ -0,0 +1 @@
+export default ['#FF0000', '#00FF00', '#0000FF', '#000000'].map(x => ({color: x}));
```

Comment:
Why does it need to be mapped to objects?

Reply:
The `checkbox-group` searches for color field in an array of objects. It supports array of strings only for labels, but not colors.